lib/deprecation-cop-view.coffee

```diff
@@ -208,27 +198,32 @@ class DeprecationCopView extends ScrollView
                         @a class:'stack-line-location', href: location, location
 
   updateSelectors: ->
-    @refreshSelectorsButton.hide()
     @selectorList.empty()
     self = this
 
-    for packageName, deprecationsByFile of getSelectorDeprecations()
+    deprecations = getSelectorDeprecations()
+
+    if Object.keys(deprecations).length == 0
       @selectorList.append $$ ->
-        @li class: 'deprecation list-nested-item collapsed', =>
-          @div class: 'deprecation-info list-item', =>
-            @span class: 'text-highlight', packageName
-
-          @ul class: 'list', =>
-            for sourcePath, deprecations of deprecationsByFile
-              @li class: 'list-item source-file', =>
-                @a class: 'source-url', href: path.join(deprecations[0].packagePath, sourcePath), sourcePath
-                @ul class: 'list', =>
-                  for deprecation in deprecations
-                    @li class: 'list-item deprecation-detail', =>
-                      @span class: 'text-warning icon icon-alert'
-                      @div class: 'list-item deprecation-message', =>
-                        @raw marked(deprecation.message)
-
-                      @div class: 'btn-toolbar', =>
-                        if url = self.createSelectorIssueUrl(packageName, deprecation, sourcePath)
-                          @a class: 'issue-url', href: url, "Create Issue on #{packageName} repo"
+        @li class: 'list-item', "No deprecated selectors"
+    else
+      for packageName, deprecationsByFile of deprecations
+        @selectorList.append $$ ->
+          @li class: 'deprecation list-nested-item collapsed', =>
+            @div class: 'deprecation-info list-item', =>
+              @span class: 'text-highlight', packageName
+
+            @ul class: 'list', =>
+              for sourcePath, deprecations of deprecationsByFile
+                @li class: 'list-item source-file', =>
+                  @a class: 'source-url', href: path.join(deprecations[0].packagePath, sourcePath), sourcePath
+                  @ul class: 'list', =>
+                    for deprecation in deprecations
+                      @li class: 'list-item deprecation-detail', =>
+                        @span class: 'text-warning icon icon-alert'
+                        @div class: 'list-item deprecation-message', =>
+                          @raw marked(deprecation.message)
+
+                        @div class: 'btn-toolbar', =>
+                          if url = self.createSelectorIssueUrl(packageName, deprecation, sourcePath)
+                            @a class: 'issue-url', href: url, "Create Issue on #{packageName} repo"
```

Comment:
I added an early return to minimize the diff a bit.

I think it flows okay to have the empty check return early since it is the "special case" in this method and it keeps the indentation level for the "main" path as low as possible.